src/app/components/FormatCredit/index.lazy.scss

```diff
@@ -1,23 +1,48 @@
 @import '../../constants';
 
 .FormatCredit {
-  color: $color-black-transparent-75;
-  font-family: var(--od-font-stack-sans);
+  color: var(--od-colour-text-primary);
+  font-family: var(--od-font-stack-serif);
   font-size: 1rem;
-  font-weight: bold;
-  line-height: 1.555555556;
-  text-transform: uppercase;
-  letter-spacing: 0.0625em;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 125%;
 
-  .u-richtext-invert & {
-    color: $color-white-transparent-75;
-  }
-
-  .FormatCredit > * {
+  .FormatCredit>* {
     color: inherit;
   }
 
-  & a {
-    white-space: nowrap;
+  // pill/tag w specificity to fight Odyssey richtext links
```

Comment:
Fixed the issue with the wrong link colour inherited from Odyssey links.

This issue comes from here: https://github.com/abcnews/odyssey/blob/68b1a7310f1897b5c45135e9502dd67c4fcf7092/src/app/components/utilities/richtext.scss#L402

I'm wondering if we should have a `:not(.custom)` in there so we can style our own UI elements without fighting specificity

Reply:
Ehhh, it can be a later problem. We don't need to solve that now, it's working.